src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportFileUploader.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import {
+  notification,
+  Space,
+  Typography,
+} from "antd";
+import { DragUpload } from "../../../../components/files/DragUpload";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+import { SampleMetadataImportSteps } from "./SampleMetadataImportSteps";
+
+export function SampleMetadataImportFileUploader(props) {
+  const { Text, Title, Paragraph } = Typography
```

Comment:
Paragraph is never used

Reply:
Please see 31084f5fe8a11ace5f6c64aa30667a9c426e406b.